src/integration-test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitIntegrationTestBase.java

```diff
@@ -35,35 +26,50 @@
 import com.scalar.db.io.Key;
 import com.scalar.db.io.Value;
 import com.scalar.db.transaction.consensuscommit.Coordinator.State;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.IntStream;
-import org.assertj.core.api.Assertions;
 
-public class ConsensusCommitIntegrationTest {
-  static final String NAMESPACE = "integration_testing";
-  static final String TABLE_1 = "tx_test_table1";
-  static final String TABLE_2 = "tx_test_table2";
-  static final String ACCOUNT_ID = "account_id";
-  static final String ACCOUNT_TYPE = "account_type";
-  static final String BALANCE = "balance";
-  static final int INITIAL_BALANCE = 1000;
-  static final int NUM_ACCOUNTS = 4;
-  static final int NUM_TYPES = 4;
-  static final String ANY_ID_1 = "id1";
-  static final String ANY_ID_2 = "id2";
-  private final ConsensusCommitManager manager;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
These static import statements (and normal ones) are probably moved by Intellij.
I ran `java -jar google-java-format-1.7-all-deps.jar ` with command-line, and they are moved back to the original location.

Reply:
Ah yes... Since I use the Google java format plugin, I thought I followed the Google style even the import order. But it looks like the import order is not handled by the plugin... I will fix the import order for the files that are changed in this PR.